android/assets/jsons/translations/German.properties

```diff
@@ -2643,8 +2643,7 @@ Mogadishu = Mogadischu
 Congratulations, conquerer. This tribe serves you now. = Glückwunsch, Eroberer. Dieser Stamm dient nun Dir.
 
 Barbarians = Barbaren
- # Requires translation!
-Can only heal by pillaging = 
+Can only heal by pillaging = Kann nur durch Plündern geheilt werden
```

Comment:
Better "Kann sich nur durch Plündern heilen"?

Reply:
You are right. Sounds much better!